src/com/duckduckgo/mobile/android/adapters/RecyclerMainFeedAdapter.java

```diff
@@ -103,9 +120,7 @@ public HeaderViewHolder(View itemView) {
             instructionbutton = (Button) itemView.findViewById(R.id.instruction_button);
             dismissImageButton = (ImageButton) itemView.findViewById(R.id.dismiss_image_button);
             viewPager = (ViewPager) itemView.findViewById(R.id.view_pager);
-            adapter = new BannerOnboardingAdapter(fragmentManager);
-            viewPager.setAdapter(adapter);
-            viewPager.setPageTransformer(false, new OnboardingTransformer());
+            viewPager.setAdapter(new BannerOnboardingAdapter());
             viewPager.addOnPageChangeListener(new ViewPager.OnPageChangeListener() {
```

Comment:
Not something you've changed here but worth mentioning. The call to `addOnPageChangeListener` on this line (and a few others in the project) is an api 9 call however our minimum api is 8. Since there are almost no devices using api 8 and google themselves are no longer supporting it, let's bump the project to 9. We'll look at stats and see if we can take this even further in our new app.